apps/nameguard.io/src/app/atoms/CalButton.tsx

```diff
@@ -20,12 +21,14 @@ export const CalButton = ({
   }, []);
 
   return (
-    <button
+    <UIButton
```

Comment:
Can you confirm I'm testing this in the right place?

![CleanShot 2024-05-21 at 19 16 54](https://github.com/namehash/nameguard/assets/126201998/70275465-00e5-4887-8b3b-5336b27279ce)

I tried hovering over this button but it seems we don't have a hover state on it anymore? I'll share more related styling feedback for our new button component in another comment.